_docs/arch_load_balanced.md

```diff
@@ -1,32 +1,32 @@
 ---
 layout: architecture
-title: Master of Masters
+title: Load Balanced
 date: 2019-07-09
 version: v0.0.1
-summary: A complete master/agent architecture with multiple compile masters and load balancing for redundancy.
+summary: A complete server/agent architecture with multiple compilers and load balancing for redundancy.
 ---
 
 ## Intended Audience
 
 This architecture is intended for large infrastructures or dynamic infrastructures
-that require the redundancy of multiple compile masters.
+that require the redundancy of multiple compilers.
 
 
 <div class="mermaid">
   graph LR;
   git(Git Repository)
   Foreman(The Foreman)
   Webhook(Puppet Webhook Server)
-  AllCM((All Compile Masters))
+  AllCM((All Compilers))
 
   PuppetDB
 
-  PuppetServerMoM{Master of Masters}
+  PuppetServerMoM{Main Puppet Server}
```

Comment:
Awesome work @binford2k 

Noticed that PuppetServerMoM is still there, perhaps `s/MoM/MPS` ? Anyhow, it may leave people scratching their head what a Puppet MoM is. Simlar thing with CM below.